test/cppsuite/test_harness/test_harness.hpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+#include <cstdlib>
+
+extern "C" {
+    #include "wiredtiger.h"
+}
+
+namespace TestHarness {
```

Comment:
You probably want to put include guards on this file to stop it from getting included multiple times.

Reply:
8fe7d6e